packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/fastlane/Fastfile

```diff
@@ -425,10 +426,10 @@ platform :ios do
             }
 
             gym(
-                workspace: PROJECT_DIRECTORY_PATH+'/ios/Runner.xcworkspace',
+                workspace: PROJECT_PATH_ABS+'/ios/Runner.xcworkspace',
```

Comment:
Please confirm this works with 

1. Local distribution
2. Github Actions
3. CodeMagic